addon/components/behave/scrollable/content.hbs

```diff
@@ -0,0 +1,9 @@
+<div
+  class="Behave-Scrollable-Content"
+  {{(if @isEnabled (modifier "on-resize" @trackSize))}}
```

Comment:
This still seems to break ember-template-lint parsing!

I guess we have to disable linting for this file, as we do here: https://github.com/kaliber5/k5win-frontend/blob/develop/.template-lintrc.js#L9-L13

Reply:
Updated.